src/test/java/cn/edu/thu/tsfile/compress/CompressTest.java

```diff
@@ -37,7 +36,7 @@ public void noCompressorTest() throws UnsupportedEncodingException, IOException
       out.write(inputString.getBytes("UTF-8"));
```

Comment:
noCompressorTest method does not throw UnsupportedEncodingException, and all "compresser"s of inputString should be "compressor"?